stdeb/util.py

```diff
@@ -257,13 +257,7 @@ def normstr(s):
 
 def get_date_822():
     """return output of 822-date command"""
-    cmd = '/bin/date'
-    if not os.path.exists(cmd):
-        raise ValueError('%s command does not exist.'%cmd)
-    args = [cmd,'-R']
-    result = get_cmd_stdout(args).strip()
-    result = normstr(result)
-    return result
+    return time.strftime("%a, %e %b %Y %H:%M:%S %z")
```

Comment:
`%e` returns the day of the month without leading zeros but it seems `data -R` includes a leading zero for single digit days. Changing this to `%d` should resolve this.

Reply:
```
$ date -R
Fri, 08 May 2020 23:19:37 +0300
```

```python
time.strftime("%a, %e %b %Y %H:%M:%S %z")
'Fri,  8 May 2020 23:20:35 +0300'
```

You are right.

Reply:
>>> time.strftime("%a, %d %b %Y %H:%M:%S %z")
'Fri, 08 May 2020 23:22:18 +0300'